rotkehlchen/assets/spam_assets.py

```diff
@@ -1041,17 +1041,25 @@
 ]
 
 
-def query_token_spam_list(db: 'DBHandler') -> set[Asset]:
+def _save_or_update_spam_assets(
+        db: 'DBHandler',
+        list_of_assets: list[dict[str, Any]],
+) -> set[Asset]:
     """
     Generate a set of assets that can be ignored using the list of spam
-    assets KNOWN_EVM_SPAM_TOKENS.
+    assets list_of_assets.
+
+    This function doesn't make any external query.
```

Comment:
nitpick: since the function name changed, I don't think this disclaimer is needed.